src/gui/properties/speedplotview.cpp

```diff
@@ -119,9 +119,9 @@ bool SpeedPlotView::Averager::push(const SampleData &sampleData)
         m_accumulator[id] += sampleData[id];
 
     // system may go to sleep, that can cause very big elapsed interval
-    const milliseconds updateInterval {static_cast<int64_t>(BitTorrent::Session::instance()->refreshInterval() * 1.25)};
-    const milliseconds maxElapsed {std::max(updateInterval, m_resolution)};
-    const milliseconds elapsed {std::min(milliseconds {m_lastSampleTime.elapsed()}, maxElapsed)};
+    const auto session = BitTorrent::Session::instance();
+    const milliseconds updateInterval {static_cast<int64_t>(session->refreshInterval() * 1.25 /* overhead */)};
```

Comment:
Please use `//` for comments and move it to a separate line or end of line. `/**/` blocks can't nest and is annoying when commenting out code to test something.

Reply:
I removed the comment, it was not in the original anyway.